src/main/java/spring/concept/controllers/HelloController.java

```diff
@@ -0,0 +1,15 @@
+package spring.concept.controllers;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ResponseBody;
+
+@Controller
+public class HelloController {
```

Comment:
It might be worth removing this controller because it wasn't on our list task, but rather like